### PR TITLE
November release notes

### PR DESCRIPTION
**Pull request by NWeber-SL**
Already reviewed by Arun.

### CODE REVIEW

File: release-notes/november-2019.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ShiftLeft November 2019 Updates
+
+**Ability to be a Member of Multiple Organizations**. ShiftLeft users can now be a member of multiple ShiftLeft organizations.
+
+## ShiftLeft Ocular
+
+**[License, Install and Update via Subscription (Beta)](../using-ocular/getting-started/installation.md)**. You can now download ShiftLeft Ocular via subscription, making it easier to add new users to, and update, ShiftLeft Ocular.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/november-2019.md]:
```suggestion
**[License, Install and Update via Subscription (Beta)](../using-ocular/getting-started/installation.md)**. You can now run ShiftLeft Ocular via the ShiftLeft CLI, making it easier to add new users to, and update, ShiftLeft Ocular.
```

Comment (NWeber-SL) [reply release-notes/november-2019.md]: Fixed in Release Notes. Should I also change the language in the Ocular Installation article from "subscription" to "CLI"? https://github.com/ShiftLeftSecurity/documentation/blob/nov-release-notes/using-ocular/getting-started/installation.md

Comment (vlad-alexandru-ionescu) [reply release-notes/november-2019.md]: yes, please

Comment (NWeber-SL) [reply release-notes/november-2019.md]: Done

File: release-notes/november-2019.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**New Language Support**. ShiftLeft Ocular now supports the examination and investigation of applications written in the following languages:
+
+* [Golang](../languages/golang.md)
+
+* [LLVM (Beta)](../languages/llvm.md) 
+
+## ShiftLeft Inspect
+
+**[Golang Support](../languages/golang.md)**. ShiftLeft Inspect now supports the analysis of applications written in Golang.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/november-2019.md]: We should mark this as "early preview".

File: using-ocular/getting-started/installation.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Refer to the article [Memory Size Recommendations](../about/ocular-memory-size.m
 
 ## Downloading and Installing ShiftLeft Ocular
 
-To download and license ShiftLeft Ocular via subscription:
+To download and license ShiftLeft Ocular via CLI:
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/installation.md]:
```suggestion
To download and license ShiftLeft Ocular via the ShiftLeft CLI:
```

Mind also adding a link on "ShiftLeft CLI" to where the user can read more about the ShiftLeft CLI?

File: using-ocular/getting-started/installation.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ Once you have installed or updated ShiftLeft Ocular, you [start ShiftLeft Ocular
 
 There are currently two methods for installing, licensing and updating ShiftLeft Ocular:
 
-* [Download via subscription (Beta)](#downloading-and-installing-shiftleft-ocular).
+* [Download via the Command Line Interface (CLI) (Beta)](#downloading-and-installing-shiftleft-ocular).
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/installation.md]:
```suggestion
* [Download via the ShiftLeft CLI (Beta)](#downloading-and-installing-shiftleft-ocular).
```

File: release-notes/november-2019.md (hunk 2 above)
Comment (NWeber-SL) [reply release-notes/november-2019.md]: Done

File: using-ocular/getting-started/installation.md (hunk 3 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/installation.md]: Done.

File: using-ocular/getting-started/installation.md (hunk 4 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/installation.md]: Done

**Pull request merged**